x-pack/plugins/maps/common/descriptor_types/sources.ts

```diff
@@ -84,7 +91,6 @@ export type KibanaTilemapSourceDescriptor = AbstractSourceDescriptor;
 
 export type WMSSourceDescriptor = AbstractSourceDescriptor & {
   serviceUrl: string;
-  layers: string;
```

Comment:
Why was layers removed from WMSSourceDescriptor? Seems this PR should not effect that type